src/Microsoft.TestPlatform.PlatformAbstractions/Microsoft.TestPlatform.PlatformAbstractions.csproj

```diff
@@ -7,12 +7,11 @@
   <PropertyGroup>
     <AssemblyName>Microsoft.TestPlatform.PlatformAbstractions</AssemblyName>
     <RootNamespace>Microsoft.TestPlatform.PlatformAbstractions</RootNamespace>
-    <TargetFrameworks>net462;netcoreapp1.0;netcoreapp2.1;netstandard1.3;netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>$(TargetNetFxVersion);$(TargetNetCoreVersion);netstandard1.3;netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
I since realized that you remove an executable target here, and not in the other case above, I still don't know how OM above builds, without platform abstractions producing netstandard1.0 compatible dll.

Reply:
Yes, I thought we wanted to remove anything (netcore) before 3.1. Isn't it the case?

Reply:
Yeah, and netstandard1.3 would imho fall into that as well. But I think we already solved this in "offline" chat.